script.js

```diff
@@ -0,0 +1,89 @@
+let playerScore = 0;
+let computerScore = 0;
+let playerChoice;
+
+function getComputerChoice() {
+  let computerChoice = Math.floor(Math.random() * 3);
```

Comment:
For a new programmer I would say this advice is very nit picky, but is helpful advice for bigger projects, probably not needed for this one.

First thing I recommend with variables as good practice is to not change the type of thing it holds through it's life cycle, in javascript you can, but I would advise against it because with maintenance it can be hard to tell what a variable should hold if you picked a random moment during that variable's life.  In this case it starts as a "number" 0, 1, or 2 and then becomes a "string" "rock", "paper" or "scissors".  One way you can avoid this is to have separate variables, e.g. a "computerChoiceNum" and "computerChoiceStr".  Another way in this current function you could actually just put "return "rock";" "return "paper";" and "return "scissors";" in each respective if block without saving them to variables, though some people don't recommend having multiple return statements.

A second thing to note, in general when people first start programming with javascript, and honestly for years after I usually see people just use "let" for every variable.  In general I try to use const where ever possible/where it makes sense to.  What const means is "this value will not be overridden" and it actually will error out if you do try to override the value anywhere in the code.  The reason this I lean towards "const" is there are no random places where the value will change and therefore as code gets to be longer and longer it generally remains more maintainable.  If values can change anywhere in your code and you have 20,000 lines of code it can become hard to track down where something goes wrong. If you separated out computerChoiceNum it could be a constant because after you generate the random number it does not change before this function completes. (each call of this function afterwards has it's own computerChoiceNum so making it const would not affect multiple calls of this function)